tests/unit/plugins/inventory/test_proxmox.py

```diff
@@ -281,6 +361,152 @@ def get_json(url):
+                            "prefix": 24,
+                            "ip-address-type": "ipv4"
+                        }
+                    ]
+                },
+                {
+                    "name": "docker0",
+                    "ip-addresses": [
+                        {
+                            "ip-address": "172.17.0.1",
+                            "prefix": 16,
+                            "ip-address-type": "ipv4"
+                        }
+                    ],
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "statistics": {
+                        "rx-errs": 0,
+                        "tx-errs": 0,
+                        "rx-packets": 0,
+                        "tx-packets": 0,
+                        "tx-dropped": 0,
+                        "rx-bytes": 0,
+                        "rx-dropped": 0,
+                        "tx-bytes": 0
+                    }
+                },
+                {
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "name": "datapath"
+                },
+                {
+                    "name": "weave",
+                    "ip-addresses": [
+                        {
+                            "ip-address": "10.42.0.1",
+                            "ip-address-type": "ipv4",
+                            "prefix": 16
+                        }
+                    ],
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "statistics": {
+                        "rx-bytes": 127289123306,
+                        "tx-dropped": 0,
+                        "rx-dropped": 0,
+                        "tx-bytes": 43827573343,
+                        "rx-packets": 132750542,
+                        "tx-packets": 74218762,
+                        "rx-errs": 0,
+                        "tx-errs": 0
+                    }
+                },
+                {
+                    "name": "vethwe-datapath",
+                    "hardware-address": "ff:ff:ff:ff:ff:ff"
+                },
+                {
+                    "name": "vethwe-bridge",
+                    "hardware-address": "ff:ff:ff:ff:ff:ff"
+                },
+                {
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "name": "vxlan-6784"
+                },
+                {
+                    "name": "vethwepl0dfe1fe",
+                    "hardware-address": "ff:ff:ff:ff:ff:ff"
+                },
+                {
+                    "name": "vethweplf1e7715",
+                    "hardware-address": "ff:ff:ff:ff:ff:ff"
+                },
+                {
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "name": "vethwepl9d244a1"
+                },
+                {
+                    "hardware-address": "ff:ff:ff:ff:ff:ff",
+                    "name": "vethwepl2ca477b"
+                }
```

Comment:
```suggestion
                },
                {
                    "name": "nomacorip",
                }
```
Test case for defaulting ip-address and mac-address.
If you have a real example of an interface where no Layer-2 address is returned feel free to replace this